### ISSUE TITLE
Bug: Assign activityiconcolorcollaboration to activityiconcolorscommunication

### ISSUE THREAD

**Issue by ekreim** [moodle-an-hochschulen/moodle-theme_boost_union#279]
Setting of `activityiconcolorcommunication` will not be assigned.

current lib.php

```php
if (get_config('theme_boost_union', 'activityiconcolorcollaboration')) {
    $activityiconcolors[] = '"communication": '.get_config('theme_boost_union', 'activityiconcolorcollaboration');
}
```

should be replaced by

```php
if (get_config('theme_boost_union', 'activityiconcolorcommunication')) {
    $activityiconcolors[] = '"communication": '.get_config('theme_boost_union', 'activityiconcolorcommunication');
}
```

**Comment by danowar2k** [comment]
I made the mistake, I'll fix it.

**Comment by danowar2k** [comment]
#279 should solve this.

**Comment by abias** [comment]
This was solved by #281. Thanks for reporting!
